cli/cmd.js

```diff
@@ -179,6 +182,69 @@ commander
 
   });
 
+commander
+  .command('upgradekvm')
+  .option('-o, --org <org>', 'the organization')
+  .option('-e, --env <env>', 'the environment')
+  .option('-u, --username <user>', 'username of the organization admin')
+  .option('-p, --password <password>', 'password of the organization admin')
+  .option('-v, --virtualhost <virtualhost>', 'virtual host of the proxy')
+  .option('-b, --baseuri <baseuri>', 'baseuri for management apis')  
```

Comment:
we should add a -r for the runtime url to access the edge auth proxy. https://domain:9001